README.md

```diff
@@ -20,6 +20,12 @@ The packages are available on npm. In the example below we use the KPN Style Bun
 npm install --save @kpn-style/bundle
 ```
 
+Or, if you are using Yarn:
```

Comment:
Maybe also add a ### yarn header and a variation of the text under the ### npm header?